service/src/main/java/com/commercetools/pspadapter/payone/IntegrationService.java

```diff
@@ -59,18 +65,32 @@ public void start() {
 
         Spark.post("/payone/notification", (req, res) -> {
             LOG.debug("<- Received POST from Payone: " + req.body());
-            final Notification notification = Notification.fromKeyValueString(req.body(), "\r?\n?&");
-
             try {
+                final Notification notification = Notification.fromKeyValueString(req.body(), "\r?\n?&");
                 notificationDispatcher.dispatchNotification(notification);
-            } catch (RuntimeException ex) {
+            } catch (Exception e) {
+                LOG.error("Payone notification handling error. Request body: " + req.body(), e);
                 res.status(400);
-                return ex.getMessage();
+                return e.getMessage();
             }
             res.status(200);
             return "TSOK";
         });
 
+        // This is a temporary jerry-rig for the load balancer to check connection with the service itself.
+        // For now it just returns a JSON response {"status":200}
+        // It should be expanded to a more real health-checker service, which really performs PAYONE status check.
+        // But don't forget, a load balancer may call this URL very often (like 1 per sec),
+        // so don't make this request processor heavy, or implement is as independent background service.
+        Route healthRoute = (req, res) -> {
+            res.status(HttpStatusCode.OK_200);
+            res.type(ContentType.APPLICATION_JSON.getMimeType());
+            return ImmutableMap.of("status", HttpStatus.OK_200);
+        };
+
+        Spark.get("/health", healthRoute, SphereJsonUtils::toJsonString);
+        Spark.get("/health/", healthRoute, SphereJsonUtils::toJsonString);
```

Comment:
done 051d05266f7b9975cb24fa4806a4b389e1a57dd7 and 927adfa637918c20feb03242242f9d57f5561669